packages/protocol/contracts/identity/Escrow.sol

```diff
@@ -117,7 +118,7 @@ contract Escrow is
   ) external nonReentrant returns (bool) {
     require(token != address(0) && value > 0 && expirySeconds > 0, "Invalid transfer inputs.");
     require(
-      !(identifier.length <= 0 && !(minAttestations == 0)),
+      !(identifier == 0 && !(minAttestations == 0)),
```

Comment:
> if you also find this way of writing it to be unnatural
yes highly lol, initially left it like this to make it easier for auditors to review? but tbh agree with you, maybe it's better to have the entire expression be more intuitive. 😅 

re: comment: do you think it'd be helpful? I only added it below bc it seems a bit quirky otherwise to check minAttestations (instead of the previous identifier check), but happy to add a comment here as well if you think it makes it more understandable! I've been looking at this too long, fresh perspective/eyes v helpful 🙏

Reply:
Thinking about this now I feel like the comment below is sufficient :)